test/test_other.py

```diff
@@ -3296,6 +3296,14 @@ def test_embind_return_value_policy(self):
 
     self.do_runf('embind/test_return_value_policy.cpp')
 
+  @parameterized({
+    '': [[]],
+    'asyncify': [['-sASYNCIFY=1']]
+  })
+  def test_embind_bigint(self, args):
+    self.do_runf('embind/test_bigint.cpp', '1000000000000n\n-1000000000000n',
```

Comment:
Is this new file missing from this PR?

Can you call it `test_embind_bigint` to match the test name?

Should it be called `test_embind_int64` perhaps?  Or `test_embind_long_long`?